### PR TITLE
tcpdf update.

### LINKED ISSUE #424: TCPDF version update
The TCPDF version used currently is 5.9.207. If the project wishes to remain in this series, then it can upgrade to the latest in it at:
http://sourceforge.net/projects/tcpdf/files/OldFiles/tcpdf_5.9.209.zip
The only changed files are:
````
CHANGELOG.TXT
composer.json
README.TXT
tcpdf.php
````

### PR DESCRIPTION
**Pull request by simonschaufi**
FIXES #424 
- replace else if with elseif
- remove include calls of tcpdf and use composer autoloader
- lots of code formatting

### CODE REVIEW

File: extensions/ki_export/export_pdf.php [hunk 1]
```diff
@@ -3,214 +3,227 @@
  * This file is part of
  * Kimai - Open Source Time Tracking // http://www.kimai.org
  * (c) 2006-2009 Kimai-Development-Team
- *
  * Kimai is free software; you can redistribute it and/or modify
```
Comment (kevinpapst) [inline extensions/ki_export/export_pdf.php:7]: Please re-add the empty lines in this file

File: extensions/ki_export/private_db_layer_mysql.php [hunk 2]
```diff
@@ -3,16 +3,13 @@
  * This file is part of
  * Kimai - Open Source Time Tracking // http://www.kimai.org
  * (c) 2006-2009 Kimai-Development-Team
- *
  * Kimai is free software; you can redistribute it and/or modify
```
Comment (kevinpapst) [inline extensions/ki_export/private_db_layer_mysql.php:7]: Please re-add the empty license lines in this file

File: extensions/ki_export/private_func.php [hunk 3]
```diff
@@ -3,28 +3,46 @@
  * This file is part of
  * Kimai - Open Source Time Tracking // http://www.kimai.org
  * (c) Kimai-Development-Team since 2006
- *
  * Kimai is free software; you can redistribute it and/or modify
```
Comment (kevinpapst) [inline extensions/ki_export/private_func.php:7]: Please re-add the empty lines in this file

File: extensions/ki_export/private_func.php [hunk 4]
```diff
@@ -43,186 +61,227 @@
...
-      $arr[$key] = null;
-    $arr['location'] = $default_location;
+function export_get_data($start, $end, $users = null, $customers = null, $projects = null, $activities = null, $limit = false, $reverse_order = false, $default_location = '', $filter_cleared = -1, $filter_type = -1, $limitCommentSize = true, $filter_refundable = -1)
+{
+    global $expense_ext_available, $database;
+    $timeSheetEntries = array();
+    $expenses = array();
+    if ($filter_type != 1) {
+        $timeSheetEntries = $database->get_timeSheet($start, $end, $users, $customers, $projects, $activities, $limit,
+            $reverse_order, $filter_cleared);
```
Comment (kevinpapst) [inline extensions/ki_export/private_func.php]: Can you check your formatter settings and allow at least 120 characters per line?
I think this line wrapping is not necessary

File: extensions/ki_export/processor.php [hunk 5]
```diff
@@ -3,16 +3,13 @@
  * This file is part of
  * Kimai - Open Source Time Tracking // http://www.kimai.org
  * (c) Kimai-Development-Team since 2006
- *
  * Kimai is free software; you can redistribute it and/or modify
```
Comment (kevinpapst) [inline extensions/ki_export/processor.php:7]: Please re-add the empty lines in this file

File: composer.json [hunk 6]
```diff
@@ -26,7 +27,7 @@
         ],
         "psr-0": {
             "Kimai_": "libraries/",
-            "TCPDF": "libraries/tcpdf/tcpdf.php"
+            "TCPDF": "libraries/tecnickcom/tcpdf/tcpdf.php"
```
Comment (kevinpapst) [inline composer.json]: This line can be removed, see the generated file libraries/composer/autoload_classmap.php
It is automatically added through the composer.json of tecnickcom/tcpdf.

After removing, call composer dump-autoload and commit the changed autoload files.

File: extensions/ki_export/private_func.php (hunk 4 above)
Comment (simonschaufi) [inline extensions/ki_export/private_func.php]: i have set 120 chars but this line is a lot longer. anyway, i changed it back by hand.

**Comment by kevinpapst** [comment]
We use tcpdf only for export, correct?

**Comment by simonschaufi** [comment]
good that you asked. it is also used in Kimai_Invoice_HtmlToPdfRenderer but not in use.

**Comment by kevinpapst** [comment]
merged last refactoring for master!

**Pull request merged**